crates/core/src/host/wasmtime/wasm_instance_env.rs

```diff
@@ -504,6 +504,42 @@ impl WasmInstanceEnv {
         })
     }
 
+    /// Starts iteration on each row, as BSATN-encoded, of a table identified by `table_id`.
+    ///
+    /// On success, the iterator handle is written to the `out` pointer.
+    /// This handle can be advanced by [`row_iter_bsatn_advance`].
+    ///
+    /// # Traps
+    ///
+    /// This function does not trap.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error:
+    ///
+    /// - `NOT_IN_TRANSACTION`, when called outside of a transaction.
+    /// - `NO_SUCH_TABLE`, when `table_id` is not a known ID of a table.
+    // #[tracing::instrument(skip_all)]
+    pub fn datastore_table_scan_bsatn(
```

Comment:
Those diffs really throw me off tbh - not much of a problem when it's 1 line, but harder to compare when it's an entire function.

Could be nice to have them in the same position just to quickly verify that functionality didn't change, only names did.

Reply:
Yeah you mentioned that on PRs with smaller changes, just saying it makes reviews (and, I imagine, Git blame) somewhat harder.